reference.conf

```diff
@@ -0,0 +1,9 @@
+typedsql {
```

Comment:
This needs to be moved to some application.conf or other config file, not the reference.conf that we ship